### DISCUSSION TITLE
Composer detected issues in your platform:

### DISCUSSION THREAD

**Discussion by cerberus478** [symfony/symfony#48741] [Q&A]
I'm getting this error
> Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.1.0". You are running 7.4.33.

but when ever I run `php -v` I get php 8.2. I'm running WAMP

**Comment by stof** [comment]
Check that your webserver runs with the same PHP version than your CLI. You might have multiple versions of PHP on your system.
